code/src/components/NoTodos.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import styled from "styled-components";
+
+export const NoTodos = () => {
+  return <TodoContainer>No To-Do's yet</TodoContainer>;
+};
+
+const TodoContainer = styled.article`
```

Comment:
Nice that you added this to the empty page!
Consider adding a prompt to add some 😄